src/lib.rs

```diff
@@ -1,13 +1,35 @@
 #[macro_use]
 extern crate include_dir;
+extern crate failure;
```

Comment:
why all those extern crate ? they are not needed anymore as of Rust 2018 no ?

Reply:
That's part of the original PR so not code of mine.

Reply:
ah yes I think we cleanup the extern crate for rust 2018 in another PR, so I think it's due to the rebase